undebt/cmd/main.py

```diff
@@ -102,23 +83,6 @@ def _find_files(paths, extensions):
         if os.path.isfile(path):
             yield path
 
-        else:
```

Comment:
If it's not a file, I think you should probably raise an error instead of silently skipping it, since we're only allowing files.